src/Picker/__tests__/index.test.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import * as React from 'react';
+import { render, shallow } from 'react-native-testing-library';
+
+import { Picker } from '..';
+
+describe('Picker', () => {
+  const testData = [{ label: 'A', value: 'a' }, { label: 'B', value: 'b' }];
+  const onValueChange = jest.fn();
+  const confirmLabel = 'OK';
+
+  const { getByText } = render(
+    <Picker
+      optionsData={testData}
+      selectedValue={testData[1].value}
+      onValueChange={onValueChange}
+      confirmLabel={confirmLabel}
+    />,
+  );
+
+  it('should contain correct label', () => {
+    expect(getByText(testData[1].label)).toBeDefined();
+  });
+
+  it('renders', () => {
```

Comment:
Sometimes I wonder of the usefulness and value of these shallow snapshots. Maybe the previous test is enough to assert that your component at least rendered the right label.